### ISSUE TITLE
Using OrbitControls  makes object clicking not work on Android (at least)

### ISSUE THREAD

**Issue by creativedrewy** [TiagoCavalcante/r3f-native-orbitcontrols#27]
I tried the native orbitcontrols sample app, and when the OrbitControls are activated clicking on the individual cubes does not work. Removing the OrbitControls from the scene entirely as well as from the `useControls()` assignment makes clicking work again.

Saw this functionality on physical Android device.

**Comment by a-jb02** [comment]
Im not sure that is correct way but these changes solved that problem

```
events: {
        ....
        onStartShouldSetResponder(event: GestureResponderEvent) {
          // On some devices this fires only for 2+ touches.
          if (!scope.enabled) return false
  
          return functions.onTouchStart(event)
        },
  
        onMoveShouldSetResponder(event: GestureResponderEvent) {
          // And on the same devices this fires only for 1 touch.
          if (!scope.enabled) return false
  
          return functions.onTouchStart(event)
        },
       ....
}
```

and here are onTouchStart changes (so you have to add moveStart: Vector3 to internals)

```
onTouchStart(event: GestureResponderEvent) {
                
        let state = event.nativeEvent.touches.length;

        if (state === STATE.ROTATE) 
        {
            const x = event.nativeEvent.touches[0].locationX;
            const y = event.nativeEvent.touches[0].locationY;
            const t = event.nativeEvent.touches[0].timestamp;

            const dx = (x - internals.moveStart.x);
            const dy = (y - internals.moveStart.y);
            const dt = (t - internals.moveStart.z);

            internals.moveStart.set(x, y, t);
            
            // prevent state changing 
            if (Math.abs(dx/dt) <= 0.5 && Math.abs(dy/dt) <= 0.5) 
            {
                state = STATE.NONE; 
            }
        }
    
        switch (state) {
          case STATE.ROTATE:
            if (!scope.enableRotate) return false;
            this.handleTouchStartRotate(event)
            internals.state = STATE.ROTATE
            return true
  
          case STATE.DOLLY:
            if (!scope.enableZoom && !scope.enablePan) return false;
            this.handleTouchStartDollyPan(event)
            internals.state = STATE.DOLLY
            return true
  
          default:
            internals.state = STATE.NONE
            return false
        }
      },
```

**Comment by TiagoCavalcante** [comment]
@creativedrewy Thanks for reporting the issue, I hadn't noted it in the devices I tested. Sorry for the delay in checking it, but it should be fixed now with 1.0.9.

@a-jb02 thanks for the code!

**Comment by creativedrewy** [comment]
Thanks for fixing this!
